[2023-06-19 11:39:15,775] ERROR log_exception() L1449 Exception on /produto [GET] - call_trace=C:\Users\thiag\OneDrive\pos-graduacao-puc\meu-mvp-sprint-1-desenv-full-stack-basico\terra_instruments_api\venv\Lib\site-packages\flask\app.py L1449
Traceback (most recent call last):
  File "C:\Users\thiag\OneDrive\pos-graduacao-puc\meu-mvp-sprint-1-desenv-full-stack-basico\terra_instruments_api\venv\Lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\thiag\OneDrive\pos-graduacao-puc\meu-mvp-sprint-1-desenv-full-stack-basico\terra_instruments_api\venv\Lib\site-packages\flask\app.py", line 1519, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\thiag\OneDrive\pos-graduacao-puc\meu-mvp-sprint-1-desenv-full-stack-basico\terra_instruments_api\venv\Lib\site-packages\flask_cors\extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
                                                ^^^^^^^^^^^^^^^^^^
  File "C:\Users\thiag\OneDrive\pos-graduacao-puc\meu-mvp-sprint-1-desenv-full-stack-basico\terra_instruments_api\venv\Lib\site-packages\flask\app.py", line 1517, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\thiag\OneDrive\pos-graduacao-puc\meu-mvp-sprint-1-desenv-full-stack-basico\terra_instruments_api\venv\Lib\site-packages\flask\app.py", line 1503, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\thiag\OneDrive\pos-graduacao-puc\meu-mvp-sprint-1-desenv-full-stack-basico\terra_instruments_api\venv\Lib\site-packages\flask_openapi3\scaffold.py", line 109, in wrapper
    resp = _do_wrapper(
           ^^^^^^^^^^^^
  File "C:\Users\thiag\OneDrive\pos-graduacao-puc\meu-mvp-sprint-1-desenv-full-stack-basico\terra_instruments_api\venv\Lib\site-packages\flask_openapi3\do_wrapper.py", line 135, in _do_wrapper
    response = func(**request_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\thiag\OneDrive\pos-graduacao-puc\meu-mvp-sprint-1-desenv-full-stack-basico\terra_instruments_api\app.py", line 94, in get_produto
    produto_id = query.id
                 ^^^^^^^^
AttributeError: 'ProdutoBuscaSchema' object has no attribute 'id'
[2023-06-19 11:39:15,791] INFO _log() L224  192.168.0.56 - - [19/Jun/2023 11:39:15] "[35m[1mGET /produto?nome=Guitarra%20Elétrica%20Gibson HTTP/1.1[0m" 500 - - call_trace=C:\Users\thiag\OneDrive\pos-graduacao-puc\meu-mvp-sprint-1-desenv-full-stack-basico\terra_instruments_api\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-06-19 11:40:24,257] ERROR log_exception() L1449 Exception on /produto [GET] - call_trace=C:\Users\thiag\OneDrive\pos-graduacao-puc\meu-mvp-sprint-1-desenv-full-stack-basico\terra_instruments_api\venv\Lib\site-packages\flask\app.py L1449
Traceback (most recent call last):
  File "C:\Users\thiag\OneDrive\pos-graduacao-puc\meu-mvp-sprint-1-desenv-full-stack-basico\terra_instruments_api\venv\Lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\thiag\OneDrive\pos-graduacao-puc\meu-mvp-sprint-1-desenv-full-stack-basico\terra_instruments_api\venv\Lib\site-packages\flask\app.py", line 1519, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\thiag\OneDrive\pos-graduacao-puc\meu-mvp-sprint-1-desenv-full-stack-basico\terra_instruments_api\venv\Lib\site-packages\flask_cors\extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
                                                ^^^^^^^^^^^^^^^^^^
  File "C:\Users\thiag\OneDrive\pos-graduacao-puc\meu-mvp-sprint-1-desenv-full-stack-basico\terra_instruments_api\venv\Lib\site-packages\flask\app.py", line 1517, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\thiag\OneDrive\pos-graduacao-puc\meu-mvp-sprint-1-desenv-full-stack-basico\terra_instruments_api\venv\Lib\site-packages\flask\app.py", line 1503, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\thiag\OneDrive\pos-graduacao-puc\meu-mvp-sprint-1-desenv-full-stack-basico\terra_instruments_api\venv\Lib\site-packages\flask_openapi3\scaffold.py", line 109, in wrapper
    resp = _do_wrapper(
           ^^^^^^^^^^^^
  File "C:\Users\thiag\OneDrive\pos-graduacao-puc\meu-mvp-sprint-1-desenv-full-stack-basico\terra_instruments_api\venv\Lib\site-packages\flask_openapi3\do_wrapper.py", line 135, in _do_wrapper
    response = func(**request_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\thiag\OneDrive\pos-graduacao-puc\meu-mvp-sprint-1-desenv-full-stack-basico\terra_instruments_api\app.py", line 94, in get_produto
    produto_id = query.id
                 ^^^^^^^^
AttributeError: 'ProdutoBuscaSchema' object has no attribute 'id'
[2023-06-19 11:40:24,261] INFO _log() L224  192.168.0.56 - - [19/Jun/2023 11:40:24] "[35m[1mGET /produto?nome=1 HTTP/1.1[0m" 500 - - call_trace=C:\Users\thiag\OneDrive\pos-graduacao-puc\meu-mvp-sprint-1-desenv-full-stack-basico\terra_instruments_api\venv\Lib\site-packages\werkzeug\_internal.py L224 
[2023-06-19 11:40:27,226] ERROR log_exception() L1449 Exception on /produto [GET] - call_trace=C:\Users\thiag\OneDrive\pos-graduacao-puc\meu-mvp-sprint-1-desenv-full-stack-basico\terra_instruments_api\venv\Lib\site-packages\flask\app.py L1449
Traceback (most recent call last):
  File "C:\Users\thiag\OneDrive\pos-graduacao-puc\meu-mvp-sprint-1-desenv-full-stack-basico\terra_instruments_api\venv\Lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\thiag\OneDrive\pos-graduacao-puc\meu-mvp-sprint-1-desenv-full-stack-basico\terra_instruments_api\venv\Lib\site-packages\flask\app.py", line 1519, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\thiag\OneDrive\pos-graduacao-puc\meu-mvp-sprint-1-desenv-full-stack-basico\terra_instruments_api\venv\Lib\site-packages\flask_cors\extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
                                                ^^^^^^^^^^^^^^^^^^
  File "C:\Users\thiag\OneDrive\pos-graduacao-puc\meu-mvp-sprint-1-desenv-full-stack-basico\terra_instruments_api\venv\Lib\site-packages\flask\app.py", line 1517, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\thiag\OneDrive\pos-graduacao-puc\meu-mvp-sprint-1-desenv-full-stack-basico\terra_instruments_api\venv\Lib\site-packages\flask\app.py", line 1503, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\thiag\OneDrive\pos-graduacao-puc\meu-mvp-sprint-1-desenv-full-stack-basico\terra_instruments_api\venv\Lib\site-packages\flask_openapi3\scaffold.py", line 109, in wrapper
    resp = _do_wrapper(
           ^^^^^^^^^^^^
  File "C:\Users\thiag\OneDrive\pos-graduacao-puc\meu-mvp-sprint-1-desenv-full-stack-basico\terra_instruments_api\venv\Lib\site-packages\flask_openapi3\do_wrapper.py", line 135, in _do_wrapper
    response = func(**request_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\thiag\OneDrive\pos-graduacao-puc\meu-mvp-sprint-1-desenv-full-stack-basico\terra_instruments_api\app.py", line 94, in get_produto
    produto_id = query.id
                 ^^^^^^^^
AttributeError: 'ProdutoBuscaSchema' object has no attribute 'id'
[2023-06-19 11:40:27,230] INFO _log() L224  192.168.0.56 - - [19/Jun/2023 11:40:27] "[35m[1mGET /produto?nome=1 HTTP/1.1[0m" 500 - - call_trace=C:\Users\thiag\OneDrive\pos-graduacao-puc\meu-mvp-sprint-1-desenv-full-stack-basico\terra_instruments_api\venv\Lib\site-packages\werkzeug\_internal.py L224 
